pages/views/_includes/navigation/source-link.njk

```diff
@@ -0,0 +1,14 @@
+{% macro ghLink(link, title) %}
+  {% if link %}
+    <div class="related-card">
+      <h2 class="related-card-heading">{{ title }}</h2>
+      <ul class="related-card-list">
+          <li class="related-card-item">
```

Comment:
It should not be list... We constantly have one item, and it is called "SOURCE" section. As for right now, it will be strange to present a list.